benchmark/projection/include/create_data.hpp

```diff
@@ -0,0 +1,168 @@
+#include <nektar_interface/projection/device_data.hpp>
+#include <random>
+#include <sycl/sycl.hpp>
```

Comment:
I'll change it but do we really need to support the cl:: version. As far as I can see from github Intel have had the sycl/sycl.hpp header since 2021 and ACPP since 2020.  I also don't like `using namespace` in headers though in this case it's probably fine.

Reply:
There was an implementation using CL/sycl.hpp and I can't remember which one it was now....